infra/vsts_agent_windows.tf

```diff
@@ -85,8 +85,8 @@ Invoke-WebRequest https://dl.google.com/cloudagents/windows/StackdriverLogging-v
 iex (New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1')
 
 # Install git, bash
-& choco install git --no-progress --yes 2>&1 | %{ "$_" }
-& choco install windows-sdk-10.1 --no-progress --yes 2>&1 | %{ "$_" }
+& choco install git --no-progress --yes 2>&1 | %%{ "$_" }
+& choco install windows-sdk-10.1 --no-progress --yes 2>&1 | %%{ "$_" }
```

Comment:
The new Terraform version has a new escape sequence `%{}`, so we now need to escape it with `%%{}`.